x/profiles/keeper/msg_server_dtag_transfers.go

```diff
@@ -116,31 +116,31 @@ func (k MsgServer) AcceptDTagTransferRequest(goCtx context.Context, msg *types.M
 
 	// Get the DTag to trade and make sure its correct
 	dTagWanted := request.DTagToTrade
-	dTagToTrade := currentOwnerProfile.DTag
+	dTagToTrade := requestReceiverProfile.DTag
 	if dTagWanted != dTagToTrade {
 		return nil, errors.Wrap(sdkerrors.ErrInvalidRequest, "the owner's DTag is different from the one to be exchanged")
 	}
 
 	// Change the DTag and validate the profile
-	currentOwnerProfile.DTag = msg.NewDTag
-	err = k.ValidateProfile(ctx, currentOwnerProfile)
+	requestReceiverProfile.DTag = msg.NewDTag
+	err = k.ValidateProfile(ctx, requestReceiverProfile)
 	if err != nil {
 		return nil, errors.Wrap(sdkerrors.ErrInvalidRequest, err.Error())
 	}
 
 	// Check for an existent profile of the receiving user
-	receiverProfile, exist, err := k.GetProfile(ctx, msg.Sender)
+	requestSenderProfile, exist, err := k.GetProfile(ctx, msg.Sender)
```

Comment:
Renaming `receiverProfile` into `requestSenderProfile` makes it more explicit that focus on sender of the transfer request, it also avoid from confusing between the DTag receiver of the message and the receiver of the transfer request (they are different).